src/Stratis.Bitcoin.Features.Api/Program.cs

```diff
@@ -46,12 +57,24 @@ public static IWebHost Initialize(IEnumerable<ServiceDescriptor> services, FullN
                         }
                     }
                 })
-                .UseStartup<Startup>()
-                .Build();
+                .UseStartup<Startup>();
 
-            host.Start();
+            var host = webHostBuilder.Build();
+                
 
+            host.Start();
+           
             return host;
         }
+
+        private static X509Certificate2 GetHttpsCertificate(ApiSettings apiSettings, ICertificateStore store)
```

Comment:
I think you should pass `apiSettings.HttpsCertificateFilePath` as a parameter instead of `apiSettings`, it'll make the method signature clearer.

Reply:
👍